docroot/profiles/custom/sitenow/sitenow.profile

```diff
@@ -583,3 +583,32 @@ function sitenow_page_attachments(array &$attachments) {
     $attachments['#attached']['library'][] = 'sitenow/admin-overrides';
   }
 }
+
+/**
+ * Implements hook_toolbar().
+ */
+function sitenow_toolbar() {
+
+  $url = Url::fromUri('https://docs.sitenow.uiowa.edu');
+
+  $items = [];
+  $items['support'] = [
+    '#type' => 'toolbar_item',
+    'tab' => [
+      '#type' => 'link',
+      '#url' => $url,
+      '#title' => 'SiteNow Help',
+      '#options' => [
+        'attributes' => [
+          'title' => t('Opens help documentation in a new window'),
+          'target' => '_blank',
```

Comment:
Similar to other toolbar items I inspected, it might be good to add `role=button` and `id="toolbar-item-sitenow-support"`

Reply:
Added the role and id attributes. Also added 'tool-bar' item class for further consistency.

Small detail--The button text is listed as "SiteNow Help" so created an id of "toolbar-item-sitenow-help." Minor, but is there a preference between 'Help' vs. 'Support?'